Complex/Complex.cs

```diff
@@ -17,7 +17,156 @@ public Complex(double re): this(re, 0)
         // probabil ca s-ar putea face cu expresii regulare. 
```

Comment:
in mod sigur se pot folosit regex. 
codul tau din ctor nu incape pe un ecran ==> "code smell"

secventa de cod imperativ care e greu de urmarit/depanat/inteles/intretinut. Peste o luna de zile vei mai sti ce ai facut acolo? Dar altii vor putea intelege codul tau?

Apreciez ca esti capabil sa scrii un astfel de cod dar trebuie sa te gandesti cum il poti simplifica. Mereu trebuie sa te gandesti cum poti scrie codul cat mai simplu, pe intelesul tuturor.